XIVSlothCombo/Combos/PvE/DRK.cs

```diff
@@ -244,16 +245,16 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                 if (LevelChecked(Delirium)
                     && IsEnabled(CustomComboPreset.DRK_ST_Bloodspiller))
                 {
-                    //Regular Bloodspiller
-                    if (GetBuffStacks(Buffs.Delirium) > 0)
+                    //Bloodspiller under Delirium
+                    if (GetBuffStacks(Buffs.Delirium) > 0
+                        || (TraitLevelChecked(Traits.EnhancedDelirium)
+                            && GetBuffStacks(Buffs.OlderDelirium) > 0))
                         return Bloodspiller;
```

Comment:
I'm not a DRK so I'm trying to follow.
I'm reading "If Scarlet Ready Delirium > 0 OR [ We meet level to do scarlet delirium checking but checking if older is > 0 ] " Looks like you got your buffs switched?

Also, what about
```
ushort DeliriumBuff = !TraitLevelChecked(Traits.EnhancedDelirium) ? Delirium : ScarletDelirium;
If GetBuffStacks(DeliriumBuff) > 0 
```

Reply:
Great catch, I forgot to restore an exclamtion mark to flip the logic after testing.
And great suggestion for making the logic a bit easier to read.
This is implemented in [b8f3b5f](https://github.com/Nik-Potokar/XIVSlothCombo/pull/1793/commits/b8f3b5f54bac07b1395ba06ebf16a1757edc8679).